### PR TITLE
feat: implement context invite & identity new command

### PR DESCRIPTION
**Pull request by fbozic**
- implement missing commands in the meroctl for the e2e tests
  -  context invite
  - identity generate
- cleanup up server admin primitives
  - force `camelCase` on all request/response types 
  - expose `new` method only on Response types, not ResponseData types
---
I'm pretty certain this will break some node-ui since I've enforced `camelCase` on all request/response types

### CODE REVIEW

**Review by chefsale — APPROVED** [review]
🚢

File: Cargo.lock
Comment (miraclx) [inline Cargo.lock:1]: weird.. no dependency changes, but we've got a massive bump here

did you `cargo update`?

File: crates/server-primitives/src/admin.rs [hunk 1]
```diff
@@ -286,116 +368,78 @@ impl JoinContextRequest {
...
+
+// -------------------------------------------- Identity API ----------------------------------------
 #[derive(Clone, Copy, Debug, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
-#[non_exhaustive]
-pub struct GetContextStorageResponseData {
-    pub size_in_bytes: u64,
+pub struct GenerateContextIdentityResponseData {
+    pub public_key: PublicKey,
+    pub private_key: PrivateKey,
```
Comment (miraclx) [inline crates/server-primitives/src/admin.rs:423]: we really shouldn't expose private keys at all through the server, I thought we could pre-save unassociated private keys to some record, say `ContextIdentity { context_id: [0;32] }` so when the invitation payload comes back, we'll have the private key, but that's not implemented yet

File: Cargo.lock
Comment (fbozic) [reply Cargo.lock:1]: nope, there were some merge conflicts so I've deleted the lock and let cargo recreate it. let me see if I can revert.

Comment (fbozic) [reply Cargo.lock:1]: reverted, c/p from master branch

File: crates/server-primitives/src/admin.rs (hunk 1 above)
Comment (fbozic) [reply crates/server-primitives/src/admin.rs:423]: follow up issue for this: https://github.com/calimero-network/core/issues/919

**Comment by MatejVukosav** [comment]
Did u test admin dashboard after changes?

**Comment by chefsale** [comment]
Needs fixing in node-ui and sdk

https://github.com/calimero-network/core/blob/master/packages/calimero-sdk/src/api/nodeApi.ts
https://github.com/calimero-network/core/blob/master/node-ui/src/api/dataSource/NodeDataSource.ts

**Pull request merged**